bin/local-env/install-wordpress.sh

```diff
@@ -7,6 +7,7 @@ set -e
 WP_DEBUG=${WP_DEBUG-true}
 SCRIPT_DEBUG=${SCRIPT_DEBUG-true}
 WEBSTORIES_DEV_MODE=${WEBSTORIES_DEV_MODE-true}
+MEDIA_TRASH=${MEDIA_TRASH-false}
```

Comment:
`false` is the default for this, right?

Reply:
You know I thought so too, but for some reason in this E2E env, it was true. I was getting test fails, if the delete link wasn't there.